java/org/contikios/cooja/plugins/RadioLogger.java

```diff
@@ -1034,7 +1034,7 @@ public void actionPerformed(ActionEvent event) {
         }
       }
     };
-    action.putValue(Action.SELECTED_KEY, selected ? Boolean.TRUE : Boolean.FALSE);
+    action.putValue(Action.SELECTED_KEY, selected ? true : false);
```

Comment:
Can't this just pass selected directly without the if?

Reply:
Yes, you are right. I update this.